im.go

```diff
@@ -22,7 +22,7 @@ type imResponseFull struct {
 
 // IM contains information related to the Direct Message channel
 type IM struct {
-	conversation
+	Conversation
```

Comment:
This shouldn't be exported. The fields within conversation are exported and can be accessed that way.